concepticondata/conceptlists/Gauchat-1925-480.tsv

```diff
@@ -0,0 +1,481 @@
+Gauchat-1925-480-57	57	the meadow	le pré	pratu	20	2819	GRASSLAND
+Gauchat-1925-480-58	58	to pass	passer	passare	20	558	PASS BY
+Gauchat-1925-480-59	59	to the cross	vers la croix	crŭce	20		
+Gauchat-1925-480-60	60	the well	le puits	pŭteu	20	954	WELL
+Gauchat-1925-480-61	61	the "chable"	le „chable“	catabolu	22		
+Gauchat-1925-480-62	62	This is (2)	C‘est	est	22	1579	BE
+Gauchat-1925-480-63	63	a place	un lieu	ūnu lŏcu	22	723	PLACE (POSITION)
+Gauchat-1925-480-64	64	dangerous	dangereux	dominiariu + ōsu	22	3466	DANGEROUS
+Gauchat-1925-480-65	65	The river	La rive	rīpa	22	666	RIVER
+Gauchat-1925-480-66	66	is narrow	est étroite	strĭcta	22	1267	NARROW
+Gauchat-1925-480-67	67	The attic	Le grenier	granariu	24		
+Gauchat-1925-480-68	68	and the "beater"	et le „battoir“	batt + atōriu	24		
+Gauchat-1925-480-69	69	are very close	sont tout près	prĕssu	24		
+Gauchat-1925-480-70	70	of the village	du village	villaticu	24	930	VILLAGE
+Gauchat-1925-480-71	71	Let's go	Allons-y	ambulemus	24		
+Gauchat-1925-480-72	72	together	ensemble	insĭmul	24	2482	TOGETHER
+Gauchat-1925-480-73	73	Be careful	Prends garde	cavica	26		
+Gauchat-1925-480-74	74	to stumble	de t‘“encoubler“	incopulare	26		
+Gauchat-1925-480-75	75	The bridge	Le pont	pŏnte	26	1011	BRIDGE
+Gauchat-1925-480-76	76	is	est en	est ĭn	26		
+Gauchat-1925-480-77	77	stone	pierre	pĕtra	26	857	STONE
+Gauchat-1925-480-78	78	iron	en fer	fĕrru	26	621	IRON
+Gauchat-1925-480-79	79	It is new	Il est neuf	nŏvu	28		
+Gauchat-1925-480-80	80	broad	large	largu	28		BROAD
+Gauchat-1925-480-81	81	The wall	Le mur	mūru	28	933	WALL (OF HOUSE)
+Gauchat-1925-480-82	82	has started	est commencé	cuminitiatu	28		
+Gauchat-1925-480-83	83	The journey	La tour	tŭrre	28	2393	JOURNEY
+Gauchat-1925-480-84	84	(has) started	(est) commencée	cuminitiata	28		
+Gauchat-1925-480-85	85	The journeys	Les tours	tŭrres	30		
+Gauchat-1925-480-86	86	have started	(sont) commencées	cuminitiatas	30		
+Gauchat-1925-480-87	87	The bull	Le bœuf	bŏve	30		BULL
+Gauchat-1925-480-88	88	has horns	a des cornes	habet cŏrn(u)as	30	1393	HORN (ANATOMY)
+Gauchat-1925-480-89	89	The ladder	L‘échelle	scala	30		LADDER
+Gauchat-1925-480-90	90	is leaned	est dressé	directiata	30		
+Gauchat-1925-480-91	91	against	contre	contra	32	2469	AGAINST
+Gauchat-1925-480-92	92	the top	le faîte	fresta	32	1753	TOP
+Gauchat-1925-480-93	93	of the roof	du toit	tēctu	32	769	ROOF
+Gauchat-1925-480-94	94	He closes	Il ferme	claudit	32	1046	SHUT
+Gauchat-1925-480-95	95	the window	la fenêtre	fenĕstra	32	1375	WINDOW
+Gauchat-1925-480-96	96	of the room	de la chambre	pē(n)sile	32	1450	ROOM
+Gauchat-1925-480-97	97	He sweeps	Il balaie	scōpat	34	230	SWEEP
+Gauchat-1925-480-98	98	in front of	devant	de + abante	34	1883	IN FRONT OF
+Gauchat-1925-480-99	99	the door	la porte	pŏrta	34	1567	DOOR
+Gauchat-1925-480-100	100	of the barn	de la grange	granica	34	3025	BARN OR SHED
+Gauchat-1925-480-101	101	We have	On a	hŏmo habet	34		
+Gauchat-1925-480-102	102	lost	perdu	perdūtu	34	1660	LOSE
```

Comment:
Isn't this the state (something is lost) instead of the action?

Reply:
should be unmapped.